webpack.karma.config.js

```diff
@@ -19,6 +19,11 @@ module.exports = (env) => {
     output: _.merge({}, baseConfig.output, {
       chunkFilename: 'javascripts/chunks/[name].bundle.js',
     }),
+    plugins: [
+      new webpack.DefinePlugin({
+        DEF_CHINA_INFRA: process.env.COCO_CHINA_INFRASTRUCTURE ? 'true' : 'false',
```

Comment:
Should these be `'true'` and `'false'` strings?  Seems like they may both evaluate to boolean true unless this is by design.

Reply:
This is by design. webpack.DefinePlugin does a direct text replacement, meaning you must provide strings. When it's compiled into `run-tests.js` it becomes a boolean.
Great pickup!